src/test/java/com/wikia/webdriver/elements/fandom/components/HeroBlock.java

```diff
@@ -19,4 +20,8 @@ public boolean isDisplayed() {
       return false;
     }
   }
+
+  public boolean hasFiveHeroUnits() {
+    return heroMosaic.findElements(By.cssSelector(".clickable-unit")).size() == 5;
```

Comment:
You could try:
```
@FindAll({@FindBy(css = "section.hero-block .hero")})
private List<WebElements> heroUnits;
```

But I think it's fine doing it this way. Only thing I would change is selecting on `.hero`, because `clickable-unit` is used for tracking and doesn't necessarily define the element itself. That CSS class could change or move to another element inside the hero unit.